docs/gathered-data.md

```diff
@@ -128,26 +128,37 @@ See: docs/insights-archive-sample/config/oauth
 Id in config: oauths
 
 
-## ClusterOperators
+## ClusterOperatorPodsAndEvents
 
-collects all ClusterOperators and their resources.
-It finds unhealthy Pods for unhealthy operators
+collects all the ClusterOperators degraded Pods
+for degraded cluster operators or that lives at the Cluster Operator's namespace, to collect:
+
+- Pod definitions
+- Previous and current Pod Container logs (when available)
+- Namespace Events
+
+* Location of pods in archive: config/pod/
+* Location of events in archive: events/
+* Id in config: operators_pods_and_events
+* Spec config for CO resources since versions:
+  * 4.6.16+
+  * 4.7+
```

Comment:
These three line shouldn't be here. It's only for the `ClusterOperators` gatherer